cpp/tests/io/parquet_test.cpp

```diff
@@ -229,6 +229,38 @@ bool read_footer(std::unique_ptr<cudf_io::datasource>& source,
   return cp.read(file_meta_data);
 }
 
+// read column index from datasource at location indicated by chunk,
+// parse and return as a ColumnIndex struct.
+// throws cudf::logic_error if the chunk data is invalid.
+cudf_io::parquet::ColumnIndex read_column_index(std::unique_ptr<cudf_io::datasource>& source,
```

Comment:
I think it would be a good policy to get away from cudf_io and just use cudf::io everywhere.

Reply:
was trying to save a char ;-)